Docker/itk-base/Dockerfile

```diff
@@ -16,7 +17,11 @@ RUN curl -O https://cmake.org/files/v3.4/cmake-3.4.3.tar.gz && \
     cd /tmp/cmake-build && \
     ../cmake-3.4.3/bootstrap && \
     make -j$(nproc) && \
-    ./bin/cmake -DCMAKE_BUILD_TYPE:STRING=Release . && \
+    cd .. && \
```

Comment:
It has been my experience that pip cmake does have SSL support. Additionally, installing it on Travis and/or CiricleCI automatically enables cache of the downloaded binary. So there may be timing benefits to using `pip install cmake` compared to downloading the basic binary.